gradle/verification-metadata.xml

```diff
@@ -82,7 +82,10 @@
          <trusted-key id="0a123c1ed3f13a6a0140e166c71fb765cd9de313" group="org.apache.ant"/>
          <trusted-key id="0cb5871fb7bf3b351614bbf6ca85ffe638d4407a" group="it.unimi.dsi" name="fastutil"/>
          <trusted-key id="0e18eae07b7774eac5db3f2113bb90ce8eafbe37" group="com.microsoft.playwright"/>
-         <trusted-key id="7b79add11f8a779fe90fd3d0893a028475557671" group="com.gradle" name="gradle-enterprise-gradle-plugin"/>
+         <trusted-key id="7b79add11f8a779fe90fd3d0893a028475557671">
+            <trusting group="com.gradle"/>
```

Comment:
🤔 We may want to use the same regex as in other entries.

This is probably sufficient for now but may need adaptations in the future.